### DISCUSSION TITLE
Suggestions needed

### DISCUSSION THREAD

**Discussion by lss381216** [Project-MONAI/model-zoo#655] [Q&A]
I am new to MONAI and please excuse me if my questions are naive. I'd like to explore denoising/edge detection/super resolution image enhancement to our images (MRI-like 2D coronal image slices) within MONAI/PyTorch architecture.  My problem is that I need to find a relevant pretrained model since we don't have the training dataset.  Any suggestions on this regard or on how to start the process are highly appreciated.  

Thank you,

**Comment by KumoLiu** [comment]
Hi @lss381216, you can find all the models here: https://monai.io/model-zoo.html.
For each of them, you can download and get some pre-trained weights. Thanks.

**Reply by lss381216** [reply]
Thank you!
